### PR TITLE
README.md: Direct link to SC 3.1.5

### PR DESCRIPTION
**Pull request by cstrobbe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ This could be useful for a lot of people:
 -	people with cognitive disabilities
 -	second language learners
 
-Among other things, it can be used to conform with [WCAG][2] 3.1.5:
+Among other things, it can be used to conform with [Success Criterion 3.1.5 in WCAG 2.0][2]:
```
Comment (xi) [inline README.md]: This should be wrapped to stay within 80 character per line.

File: README.md [hunk 2]
```diff
@@ -36,4 +36,4 @@ adoption.  This extension is merely a proof of concept. I would like to see
 this kind of functionality directly in browsers one day.
 
 [1]: http://permalink.gmane.org/gmane.ietf.languages/11065
-[2]: https://www.w3.org/TR/WCAG20/
+[2]: https://www.w3.org/TR/2008/REC-WCAG20-20081211/#meaning-supplements
```
Comment (xi) [inline README.md]: Is there any harm in linking to <https://www.w3.org/TR/WCAG20/#meaning-supplements>? I do not expect breaking changes there.

**Review by xi — CHANGES_REQUESTED** [review]
Thanks for the contribution! I have just some stylistic remark.

File: README.md (hunk 2 above)
Comment (cstrobbe) [reply README.md]: Linking directly to https://www.w3.org/TR/WCAG20/#meaning-supplements should be fine. Any changes to the SC would end up in WCAG 2.1.

**Comment by xi** [comment]
Thanks

**Review by xi — APPROVED** [review]

**Pull request merged**